firedrake/interpolation.py

```diff
@@ -285,15 +285,15 @@ def _interpolator(V, tensor, expr, subset, arguments, access):
     if not isinstance(expr, firedrake.Expression):
         kernel = compile_expression_dual_evaluation(expr, to_element,
                                                     domain=source_mesh,
-                                                    parameters=parameters,
-                                                    coffee=False)
+                                                    parameters=parameters)
```

Comment:
You need to make this change in `/firedrake/preconditioners/pmg.py` too

Reply:
Done.